astroid/helpers.py

```diff
@@ -83,6 +83,38 @@ def object_type(node, context=None):
     return list(types)[0]
 
 
+def object_isinstance(node, class_or_seq, context=None):
+    """Check if a node 'isinstance' any node in class_or_seq
+
+    :param node: A given node
+    :param class_or_seq: Union[Nodes.NodeNG], Sequence[nodes.NodeNG]]
+    :rtype: Union[bool, util.Uninferable]
+
+    :raises AstroidTypeError: if the given ``classes_or_seq`` are not types
+    """
+    if not isinstance(class_or_seq, (tuple, list)):
+        class_seq = (class_or_seq,)
+    else:
+        class_seq = class_or_seq
+    obj_type = object_type(node, context)
+    if obj_type is util.Uninferable:
+        return util.Uninferable
```

Comment:
I see that this returns boolean but also an Uninferable object, maybe we could either raise an exception here, that the caller could handle, or we could hardcode the meaning of Uninferable into a `None`. I would prefer the first one since it seems more obvious. Nevertheless, it's a nitpick.

Reply:
So I am trying to make these helpers consistent. `object_type` also returns Uninferable

Reply:
Hmm but I already raise an exception

Reply:
Yeah, it's fine if `object_type` already does that. It was just a small pet peeve regarding two different return types that should have been handled by `object_type` as well when it was introduced.